### PR TITLE
DIRECTOR: Revert Palette DO NOT MERGE

### PR DESCRIPTION
**Pull request by rvanlaar**
Palettes colors range from 0-255.
Sprite colors go from 255-0, defined colors in lingo from 0-255.
Set the color palettes in the order 0-255.

This commit reverses the palettes, removes the reversal on 'lingo' colors,
such as the stageColor.

There is work left to be done:
- A few images in SS warlock are reversed
    - The first planet that is shown after starting the game.
    - the dissappearing planet to the right of the text, just before the hands show in the opening movie.
 
- The MSRLEDecoder::decode8 is patched, this wreaks havoc for all Scummvm in general. 

@sev- 
Sprites on disk use the reverse palette.
I see three solutions:

1. Convert sprites colors while reading the file
1. Convert internal colors, i.e. Lingo colors to use a reverse palette
1. Blit operators reverse based on whether it's a sprite or not.

What is your idea on how to solve this?

### CODE REVIEW

**Comment by sev-** [comment]
Could you please be a bit more specific, which images in Warlock are reversed?

Also, did you test the Running Boy?

**Comment by rvanlaar** [comment]
The pull request comment was updated to show which images.
And Running boy has lots of problems.

File: engines/director/frame.cpp [hunk 1]
```diff
@@ -981,34 +981,39 @@ void Frame::inkBasedBlit(Graphics::ManagedSurface &targetSurface, const Graphics
...
-		drawGhostSprite(targetSurface, spriteSurface, drawRect);
-		break;
-	case kInkTypeReverse:
-		drawReverseSprite(targetSurface, spriteSurface, drawRect);
-		break;
-	default:
-		warning("Frame::inkBasedBlit(): Unhandled ink type %d", ink);
-		targetSurface.blitFrom(spriteSurface, Common::Point(drawRect.left, drawRect.top));
-		break;
+		case kInkTypeCopy:
```
Comment (sev-) [inline engines/director/frame.cpp]: The indentation is now wrong for the whole block.

Comment (rvanlaar) [reply engines/director/frame.cpp]: indeed.

**Comment by rvanlaar** [comment]
Close this one. It's handled by: https://github.com/scummvm/scummvm/pull/2161

**Pull request closed without merging**